From df6ab6ff44e89b87e1dd3db75f22900bfc547aa0 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Tue, 18 Nov 2008 17:12:15 -0800
Subject: [PATCH 13/15] Staging: comedi: dt9812: remove #ifdef that is not needed

We don't need to set the owner field of the usb_driver structure
in the dt9812 driver.

Cc: Anders Blomdell <anders.blomdell@control.lth.se>
Cc: David Schleef <ds@schleef.org>
Cc: Frank Mori Hess <fmhess@users.sourceforge.net>
Cc: Ian Abbott <abbotti@mev.co.uk>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/comedi/drivers/dt9812.c |    3 ---
 1 file changed, 3 deletions(-)

--- a/drivers/staging/comedi/drivers/dt9812.c
+++ b/drivers/staging/comedi/drivers/dt9812.c
@@ -882,9 +882,6 @@ static void dt9812_disconnect(struct usb
 }
 
 static struct usb_driver dt9812_usb_driver = {
-#ifdef COMEDI_HAVE_USB_DRIVER_OWNER
-	.owner = THIS_MODULE,
-#endif
 	.name = "dt9812",
 	.probe = dt9812_probe,
 	.disconnect = dt9812_disconnect,
